src/Store.java

```diff
@@ -0,0 +1,168 @@
+                } else {
+                    change = change - theBill.Nominal * amountOfTheBills;
+                    theBill.Amount -= amountOfTheBills;
+                }
+            } else if (theBill.Amount != 0) {
+                change = change - theBill.Nominal * amountOfTheBills;
+                theBill.Amount -= amountOfTheBills;
+            }
+        }
+        if (change == 0) {
+            String localReport = makeGiveReport(Bills, BillsCopy);
+            System.out.println(localReport);
+            Bills = BillsCopy;
+            addToLog(" Given " + amount + " UAH to a user");
+        } else {
+            System.out.println("There are no enough Bills !");
+        }
+    }
+
+    /**
+     * @param srcArray array contains information about different ATM bills before operation
+     * @param newArray array contains information about different ATM bills after operation
+     * @return string the contains information about how much and what nominal where given
+     */
+    private static String makeGiveReport(Bill[] srcArray, Bill[] newArray) {
+        StringBuilder report = new StringBuilder("");
+        for (int i = 0; i < srcArray.length; i++) {
+            int diff = srcArray[i].Amount - newArray[i].Amount;
+            if (diff > 0) {
+                String logRow = " " + diff + " bills of " + srcArray[i].Nominal + ";";
+                report.append(logRow);
+            }
+        }
+        return "Were given: " + report.toString();
+    }
+
+    /**
+     * @return the value which show if there is bills in the ATM
+     */
+    public static boolean checkBills() {
+        boolean thereIsMoney = false;
+        for (Bill bill : Bills) {
+            if (bill.Amount != 0) {
+                thereIsMoney = true;
+                break;
+            }
+        }
+        return thereIsMoney;
+    }
+
+    /**
+     * @param nominal number nominal value of the bill
+     */
+    public static void addBill(int nominal) {
+        for (int i = 0; i < 6; i++) {
+            if (Bills[i].Nominal == nominal) {
+                Bills[i].Amount += 1;
+            }
+        }
+        addToLog(" Was added " + nominal + " UAH to the ATM");
+    }
+
+    public static void printStat() {
+        if (operationsLog.size() < 1) {
+            System.out.println(">> There are no records yet");
+            return;
+        }
+        for (String bill : operationsLog) {
+            System.out.println(">> " + bill);
+        }
+    }
+
+    public static void printCash() {
+        for (Bill bill : Bills) {
+            System.out.println(">> Bills of nominal " + bill.Nominal + " - " + bill.Amount);
+        }
+    }
+
+    /**
+     * @param strategyVariant number associated with the specified strategy of giving money
+     */
+    public static void changeStrategy(int strategyVariant) {
+        strategy = strategyVariant;
+    }
+
+    public static class Bill {
+        public int Nominal;
```

Comment:
1) Field names are in camel-case starting with a small letter
2) By convention, you need to add getters